simulation_ws/src/cloudwatch_simulation/launch/turtlebot3_navigation.launch

```diff
@@ -1,10 +1,10 @@
-<!-- Copied from turtlebot3_navigation, allow initial pose arguments -->                                                                                                            
```

Comment:
this looks like a 1:1 copy of [`turtlebot3_navigation/launch/turtlebot3_navigation.launch`](https://github.com/ROBOTIS-GIT/turtlebot3/blob/master/turtlebot3_navigation/launch/turtlebot3_navigation.launch)? 

do we need a copy of this in `cloudwatch_simulation` package? why can't we reference directly to the original launch file?

Reply:
I believe as documented in the launch file, we want to have configurable initial pose which is not present in the turtlebot3_navigation package